### PR TITLE
Migrate from Travic-CI to AppVeyor

### PR DESCRIPTION
**Pull request by skubarenko**

### CODE REVIEW

**Comment by maxima-net** [comment]
This will fix #38

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+version: "#{build}"
+clone_depth: 1
+
+branches:
+  only:
+    - master
+    - dev
```
Comment (maxima-net) [inline appveyor.yml:7]: Doesn't this set ignore other branches?

**Review by maxima-net — CHANGES_REQUESTED** [review]

File: appveyor.yml (hunk 1 above)
Comment (skubarenko) [reply appveyor.yml:7]: Yes, it only runs on a PR with the dev and other branch

Comment (maxima-net) [reply appveyor.yml:7]: Ok, I see now

**Review by maxima-net — APPROVED** [review]

**Pull request merged**